libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/resolveDashboardConfig.ts

```diff
@@ -166,6 +176,7 @@ export function* resolveDashboardConfig(
         disableDefaultDrills: config.disableDefaultDrills ?? false,
         enableFilterValuesResolutionInDrillEvents: config.enableFilterValuesResolutionInDrillEvents ?? false,
         menuButtonItemsVisibility: config.menuButtonItemsVisibility ?? {},
+		allowInProgressFeatures,
```

Comment:
Strange formatting, it might trip up validation.

Reply:
fixed